sources/packages/backend/apps/queue-consumers/test/helpers/mock-utils/cas-service.mock.ts

```diff
@@ -0,0 +1,28 @@
+import { CASService } from "@sims/integrations/cas/cas.service";
+import { createFakeCASSupplierResponse } from "../../../src/processors/schedulers/cas-integration/_tests_/cas-response.factory";
```

Comment:
I do not see a problem moving `createFakeCASSupplierResponse` file to this same folder.